geodocker-geoserver/Makefile

```diff
@@ -23,10 +23,13 @@ dockerfile:
 	mkdir -p target
 	sed 's/__TAG__/'"${TAG}"'/' Dockerfile.template > target/Dockerfile
 
-tarball:
+${GEOMESA_ACCUMULO_TARBALL}:
 	mkdir -p tarballs
-	cp ${HOME}/.m2/repository/org/locationtech/geomesa/geomesa-accumulo-dist_2.11/${GEOMESA_VERSION}/${GEOMESA_ACCUMULO_TARBALL} tarballs/.
-	cp ${HOME}/.m2/repository/org/locationtech/geomesa/geomesa-kafka-dist_2.11/${GEOMESA_VERSION}/${GEOMESA_KAFKA_TARBALL} tarballs/.
+	cp ${HOME}/.m2/repository/org/locationtech/geomesa/geomesa-accumulo-dist_2.11/${GEOMESA_VERSION}/${GEOMESA_ACCUMULO_DIST} tarballs/.
+
+${GEOMESA_KAFKA_TARBALL}:
+	mkdir -p tarballs
+	cp ${HOME}/.m2/repository/org/locationtech/geomesa/geomesa-kafka-dist_2.11/${GEOMESA_VERSION}/${GEOMESA_KAFKA_DIST} tarballs/.
```

Comment:
I'm not sure if it's better to download this geomesa-accumulo and geomesa-kafka distributions from a publicly available location (e.g. [github releases](https://github.com/locationtech/geomesa/releases)), or from a local build.
@elahrvivaz do you have a preference?